packages/devtools_app/lib/src/framework/release_notes/release-notes-next.md

```diff
@@ -6,7 +6,8 @@ This is draft for future release notes, that are going to land on
 Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
 
 ## General updates
-TODO: Remove this section if there are not any general updates.
+
+- Fix an issue in Safari (browsers that do not support RegExp negative lookbehind) that prevented DevTools from loading.
```

Comment:
can you add a link to the PR?

Reply:
Done! I copied format from https://github.com/flutter/devtools/blob/master/CHANGELOG.md (PR link at the end).

I also added another PR I saw I'd missed that made some fixes to syntax highlighting.